### PR TITLE
SMS-6281: Verify Service support in DotNet SDK

### PR DESCRIPTION
**Pull request by rajneeshkatkam-plivo**
https://plivo-team.atlassian.net/browse/SMS-6281

### CODE REVIEW

File: src/Plivo/Resource/VerifySession/VerifySessionInterface.cs [hunk 1]
```diff
@@ -0,0 +1,317 @@
...
+        /// </summary>
+        /// <returns>The create.</returns>
+        /// <param name="recipient">Recipient.</param>
+        /// <param name="app_uuid">App UUID.</param>
+        /// <param name="channel">Channel.</param>
+        /// <param name="url">URL.</param>
+        /// <param name="method">Method.</param>
+        /// 
+
+        public VerifySessionCreateResponse Create(
```
Comment (narayana-plivo) [inline src/Plivo/Resource/VerifySession/VerifySessionInterface.cs:37]: why recipient has default null value?

File: src/Plivo/Resource/VerifySession/VerifySessionInterface.cs [hunk 2]
```diff
@@ -0,0 +1,317 @@
...
+        /// </summary>
+        /// <returns>The create.</returns>
+        /// <param name="recipient">Recipient.</param>
+        /// <param name="app_uuid">App UUID.</param>
+        /// <param name="channel">Channel.</param>
+        /// <param name="url">URL.</param>
+        /// <param name="method">Method.</param>
+        /// 
+        public async Task<VerifySessionCreateResponse> CreateAsync(
+            string recipient=null, string app_uuid = null, string channel = null, string url = null,
```
Comment (narayana-plivo) [inline src/Plivo/Resource/VerifySession/VerifySessionInterface.cs:73]: same here

**Review by narayana-plivo — APPROVED** [review]
LGTM

File: src/Plivo/Resource/VerifySession/VerifySessionInterface.cs (hunk 2 above)
Comment (rajneeshkatkam-plivo) [reply src/Plivo/Resource/VerifySession/VerifySessionInterface.cs]: Removed null default for recipient

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,5 +1,13 @@
 # Change Log
 
+
+## [5.38.0](https://github.com/plivo/plivo-dotnet/tree/v5.38.0) (2023-11-08)
```
Comment (narayana-plivo) [inline CHANGELOG.md:4]: can we add more detail here

**Pull request merged**